casts/free-podcasts-screencasts-en.md

```diff
@@ -194,6 +194,11 @@
 * [The EmberMap Podcast](https://embermap.com/podcast) - Sam Selikoff and Ryan Toronto (podcast)
 
 
+#### Machine Learning
```

Comment:
Remember add entry to Index/TOC listing

Reply:
A bit of help with more new resources could fit in this topic:

- `High-performance computing and AI podcast` (https://developer.ibm.com/podcasts/high-performance-computing-and-ai-podcast/) by IBM
- `Machine learning` (https://anchor.fm/david-nishimoto) by  David Nishimoto
- `Concerning AI` (https://concerning.ai) by Brandon Sanders, Ted Sarvata
- `The AI Podcast` (https://blogs.nvidia.com/ai-podcast/) by NVIDIA, Noah Kravitz

I believe that current in `Data Science` could be moved too: 
- `Talking Machines`
- `Learning Machines 101` by Richard M. Golden

Reply:
yes, I think there are enough for a new category